bundle/src/files.rs

```diff
@@ -39,6 +40,8 @@ pub struct FileSet {
     pub file_set_type: FileSetType,
     pub files: Vec<BundledFile>,
     pub glob: String,
+    /// Added in v0.6.11. Populated when parsing from BEP, not from junit globs
+    pub test_runner_status: Option<JunitReportStatus>,
```

Comment:
Nit: Name. This isn't really for a test runner, but rather the build system around it. Test runner integrations aren't going to use junit.